README.md

```diff
@@ -180,6 +180,7 @@ jobs:
 - name: Checkout submodules
   shell: bash
   run: |
+    git config --global url."https://github.com/".insteadOf "git@github.com:"
```

Comment:
Can you provide a comment to make it clear this this line is only necessary if you use ssh for local development?